tests/test_warpedvrt.py

```diff
@@ -93,24 +93,25 @@ def test_warped_vrt_add_alpha(dsrec, path_rgb_byte_tif):
 
 def test_warped_vrt_msk_add_alpha(path_rgb_msk_byte_tif, caplog):
     """Add an alpha band to the VRT to access per-dataset mask of a source"""
-    with rasterio.open(path_rgb_msk_byte_tif) as src:
-        vrt = WarpedVRT(src, crs=DST_CRS, add_alpha=True)
-        assert vrt.src_nodata is None
-        assert vrt.dst_nodata is None
-        assert vrt.count == 4
-        assert vrt.mask_flag_enums == (
-            [MaskFlags.per_dataset, MaskFlags.alpha],
-        ) * 3 + (
-            [MaskFlags.all_valid],
-        )
+    with rasterio.Env(GDAL_DISABLE_READDIR_ON_OPEN="TRUE"):
```

Comment:
The tests were failing locally because I have `GDAL_DISABLE_READDIR_ON_OPEN` set to `empty_dir` to avoid reading external files 😄 I think adding the Env here doesn't cost much and will make sure tests pass on all environments